### ISSUE TITLE
 Switching tabs or loading layouts with floating anchorables results in a nullpointerexception

### ISSUE THREAD

**Issue by scorixear** [xceedsoftware/wpftoolkit#1532]
I am currently developing a small WPF Gui and I wanted to use your proposed AvalonDock. I am currently using the version 3.5 of your Extended WPF Toolkit. Unfortunately after implementing some small functionalities which simply change the tab of a WPF tab control, I get a NullPointerException in your code referencing the Xceed.Wpf.AvalonDock.dll!Xcee.Wpf.Avalondcok.Controls.LayoutAnchorableFloatingWindowControl.OnInitialized(System.EventArgs e) Line 123.
On further exploring this issue I noticed that the same Exception happens while deserializing a layout, that contains FloatingAnchorables.

I attached my ExceptionWindow.xaml which contains the test code that produces this exception. To reproduce this error, you have to use the open source version 3.5.0, start the program with this window as main window, drag on of the "Window 1", "Window 2" or "Window 3" out of the Main Window (so that it is floating). Then click on Tab 2 (the floating window should have disappeard) and click back on Tab 1.

Best regards
Paul

```xaml
<Window xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
        xmlns:xcad="http://schemas.xceed.com/wpf/xaml/avalondock" x:Class="ExceptionWindow.ExceptionWindow"
        mc:Ignorable="d"
        Title="Exception Window" Height="800" Width="1200">

    <Grid>

        <TabControl SelectedIndex="0">
            <TabItem Header="Tab 1">
                <xcad:DockingManager x:Name="dockingManaer1" AllowMixedOrientation="True">

                    <xcad:LayoutRoot>

                        <xcad:LayoutPanel Orientation="Horizontal">


                            <xcad:LayoutAnchorablePaneGroup Orientation="Vertical" DockWidth="40*">

                                <xcad:LayoutAnchorablePane>

                                    <xcad:LayoutAnchorable ContentId="Window1" Title="Window 1" CanHide="False" CanAutoHide="False" CanFloat="True">

                                        <Grid></Grid>
                                    </xcad:LayoutAnchorable>
                                </xcad:LayoutAnchorablePane>

                                <xcad:LayoutAnchorablePane>
                                    <xcad:LayoutAnchorable ContentId="Window2" Title="Window 2" CanAutoHide="False" CanHide="True" CanClose="True">
                                        <Grid></Grid>
                                    </xcad:LayoutAnchorable>
                                </xcad:LayoutAnchorablePane>
                            </xcad:LayoutAnchorablePaneGroup>

                            <xcad:LayoutAnchorablePane DockWidth="60*">
                                <xcad:LayoutAnchorable ContentId="Window3" Title="Window 3" CanHide="True" CanAutoHide="False" CanClose="True">
                                    <Grid></Grid>
                                </xcad:LayoutAnchorable>
                            </xcad:LayoutAnchorablePane>

                        </xcad:LayoutPanel>
                    </xcad:LayoutRoot>
                </xcad:DockingManager>
            </TabItem>
            <TabItem Header="Tab 2">
                <xcad:DockingManager x:Name="dockingManager2" AllowMixedOrientation="True">
                    <xcad:LayoutRoot>
                        <xcad:LayoutPanel Orientation="Horizontal">
                            <xcad:LayoutAnchorablePane DockWidth="25*">
                                <xcad:LayoutAnchorable CanClose="False"  IsActive="True" CanHide="False" ContentId="Window4" Title="Window 4">
                                   <Grid></Grid>
                                </xcad:LayoutAnchorable>
                            </xcad:LayoutAnchorablePane>
                            <xcad:LayoutAnchorablePane DockWidth="75*">
                                <xcad:LayoutAnchorable CanClose="False" CanHide="False" CanAutoHide="False" ContentId="Window5" Title="Window 5">
                                   <Grid></Grid>
                                </xcad:LayoutAnchorable>
                            </xcad:LayoutAnchorablePane>
                        </xcad:LayoutPanel>
                    </xcad:LayoutRoot>
                </xcad:DockingManager>
            </TabItem>
        </TabControl>
    </Grid>
</Window>
```

```c#
using System.Windows;

namespace ExceptionWindow
{
    /// <summary>
    /// Interaction logic for FrmMain.xaml
    /// </summary>
    public partial class ExceptionWindow : Window
    {
        
    }
}
```
[ExceptionWindow.zip](https://github.com/xceedsoftware/wpftoolkit/files/3601166/ExceptionWindow.zip)

**Comment by Dirkster99** [comment]
The [solution to this issue](https://github.com/Dirkster99/AvalonDock/commit/59b687385df6a4328042079fce4b4046f67d4a5c) is part of the next release [AvalonDock 3.5.13 ](https://github.com/Dirkster99/AvalonDock)

**Comment by XceedBoucherS** [comment]
Hi,
Thank you for great explanation.
This issue is already fixed in v3.6, which is currently available on this site.
Thank you.
